code/src/App.js

```diff
@@ -1,9 +1,22 @@
-import React from 'react'
+import React from 'react';
+import { Provider } from 'react-redux';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+
+import todos from './reducers/todos';
+import TodoList from './components/TodoList';
+
+const reducer = combineReducers({
+  todos: todos.reducer
+});
+
+const store = configureStore({ reducer });
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
```

Comment:
you have only one immediate child (`Provider`) here sou you shouldn't use <></> here.